### DISCUSSION TITLE
Welcome to Discussion About AutoDocX!

### DISCUSSION THREAD

**Discussion by SHAYANZAWAR** [SHAYANZAWAR/AutoDocX#3] [Announcements]
## 👋 Welcome Peers!
  We’re using Discussions as a place to connect with others. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with others.
  * Welcome others and are open-minded.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community, let's go!

**Comment by hurairamuzammal** [comment]
I am Huraira Muzammal and I would like to add on to this Project for any further updates/enhancements or errors!

**Reply by SHAYANZAWAR** [reply]
I welcome you warmly and would highly appreciate you for your contributions.
